idapal.py

```diff
@@ -337,3 +337,24 @@ def unload_plugin():
     if  hooks is not None:
         hooks.unhook()
         hooks = None
+
+# Minimal Plugin Structure Stub
+class aiDAPalPlugin(idaapi.plugin_t):
+    flags = 0
```

Comment:
```suggestion
    flags = idaapi.PLUGIN_MULTI | idaapi.PLUGIN_MOD
```
> - PLUGIN_MULTI: Recommended for all plugins; this flag enables the plugin to run simultaneously across multiple opened IDBs within the same IDA instance.
> - PLUGIN_MOD: The plugin modifies the IDA database.

Not a plugin dev, just checked [the docs](https://docs.hex-rays.com/9.0/developer-guide/idapython/how-to-create-a-plugin#overview-of-myplugin-class-attributes) and these two seemed appropriate :)